packages/math-input/src/components/keypad-button.tsx

```diff
@@ -102,7 +101,7 @@ class KeypadButton extends React.PureComponent<Props> {
         // object. This method must be called whenever a property that
         // influences the possible outcomes of `this._getFocusStyle` and
         // `this._getButtonStyle` changes (such as `this.buttonSizeStyle`).
-        for (const type of Object.keys(KeyTypes)) {
+        for (const type of Object.values(KeyType)) {
```

Comment:
I verified this is correct in a [Playground](https://www.typescriptlang.org/play?ts=5.0.4#code/KYOwrgtgBAwg9gGzgJygbwFBW1ASsAEygF4oAiXAUQBEyAaLHAcWWFBPKasoDl6MAvhgwAzFFAAUAYzggAzgBcoUqHBFQA8gCMAVsCkKAdAGtgATzkT4SZAEpb6RthnzEwQ0gDm024OFjUaVlFZVV1bT0DQwA3AEMEMGBLaxR7RxxlYLcPOG8pXwEgA).